src/pages/sponsors.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { Helmet } from "react-helmet"
+import styles from "../styles/sponsors.module.css"
+import { graphql } from 'gatsby'
+
+const SponsorsPage = ( {data} ) => {
+
+	const getImageUrl = (companyName) => {
+		const URLRegex = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[\-;:&=\+\$,\w]+@)?[A-Za-z0-9\.\-]+|(?:www\.|[\-;:&=\+\$,\w]+@)[A-Za-z0-9\.\-]+)((?:\/[\+~%\/\.\w\-_]*)?\??(?:[\-\+=&;%@\.\w_]*)#?(?:[\.\!\/\\\w]*))?)/ ;
+		return URLRegex.test(companyName) ? companyName : 'https://res.cloudinary.com/chicagojs/image/upload/2018Sponsors/' + companyName;
+	}
+
+	const grabTieredSponsors = (tier) => {
+		const sponsorsArray = data.allSponsorsDataJson.edges.find(data => data.node.title === tier);
+		if(sponsorsArray.node.items.length > 0) {
+			return sponsorsArray.node.items.map((i,index) => (
+				<div className={`col-md-6 col-lg-3 ${styles.sponsorCard} ${tier === 'gold' ? 'col-lg-4' : 'col-lg-2'}`}>
+					<div className={`card border-0 transform-on-hover ${styles.cardImgContainer}`}>
+						<a className="lightbox" href={i.url} target="_blank"> 
+							<img style={[{width: tier === 'gold' ? '100%' : '75%'},styles.mobileLogo]} src={getImageUrl(i.image)} alt={`${i.name}`} className="card-img-top" />
+						</a>
+					</div>
+				</div>
+			));
+		}
+	}
+
+	return (
+		<div className="page-container">
+			<Helmet>
+					<title>Sponsors</title>
+					<meta name="description" content="Sponsors for ChicagoJS" />
+					<meta name="keywords" content="Sponsors, ChicagoJS, Javascript,Js" />
+					<meta name="author" content="Sebastian Kurpiel" />
+					{/* Bootstrap style sheet */}
+					<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous"></link>
+					<link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet"></link>    
+			</Helmet>
+			<div className={`jumbotron jumbotron-fluid ${styles.jumbotron}`}>
+					<div className="container">
+							<div className="row justify-content-center">
+									<h1 className={styles.heading}>Sponsors</h1>
+							</div>
+					</div>
+			</div>
+			<div className="container">
+					<div className="row text-center">
+							<div className="heading">
+									<p className={styles.blurb}>
+										All at ChicagoJS would like to thank our sponsors for their generosity! Without you guys none of this would
+										be possible.
+									</p>
+							</div>
+					</div>
+			</div>
+			<div className={`gallery-block `}>
```

Comment:
what do you think of this?

sponsorsArray (rename to sponsors) sponsorsArray->sponsors

```js
sponsors.map(level => <SponsorRow {...level} />

// ...

const SponsorRow = ({ title, items: sponsors }) => {
  let className = title === "gold" ? "col-6 col-md-4" : "col-4"
  return (
     <div className="SponsorRow">
      <span className="SponsorRow-title mb-2">{title}</span>
      <div className="row">
      {sponsors.map(sponsor => <a href={}><div className={className} /* or image w/e you have> */</a>
      </div>
     </div>
  )
}
```